daisy_workflows/image_import/ubuntu/translate.py

```diff
@@ -45,11 +45,17 @@
 ConnectPort 563
 '''
 
-partner_list = '''
+partner_repo = '''
 # Enabled for Google Cloud SDK
-deb http://archive.canonical.com/ubuntu {ubu_release} partner
+deb http://archive.canonical.com/ubuntu {release} partner
```

Comment:
Is this ubuntu_release param from the docs at the top of the file?

Reply:
Good question. The comment at the top is referring to the interface exposed via instance metadata. Looking at that, along with your next comment, I'm going to consolidate all three into the same variable name for clarity. Good find!